examples/large_models/Huggingface_accelerate/llama2/sample_text.txt

```diff
@@ -0,0 +1 @@
+How are you
```

Comment:
For chat example use sample similar to [chats.json](https://github.com/facebookresearch/llama-recipes/blob/main/inference/chats.json)